pkg/controller/container-runtime-config/helpers.go

```diff
@@ -437,13 +437,13 @@ func createCRIODropinFiles(cfg *mcfgv1.ContainerRuntimeConfig) []generatedConfig
 			klog.V(2).Infoln(cfg, err, "error updating user changes for log-size-max to crio.conf.d: %v", err)
 		}
 	}
-	if ctrcfg.DefaultRuntime != mcfgv1.ContainerRuntimeDefaultRuntimeEmpty {
-		tomlConf := tomlConfigCRIODefaultRuntime{}
-		tomlConf.Crio.Runtime.DefaultRuntime = string(ctrcfg.DefaultRuntime)
-		generatedConfigFileList, err = addTOMLgeneratedConfigFile(generatedConfigFileList, CRIODropInFilePathDefaultRuntime, tomlConf)
-		if err != nil {
-			klog.V(2).Infoln(cfg, err, "error updating user changes for default-runtime to crio.conf.d: %v", err)
-		}
+
+	ctrcfg.DefaultRuntime = mcfgv1.ContainerRuntimeDefaultRuntimeDefault
```

Comment:
we need to check if crun was already set, and not do this if so. we don't want to clobber